src/gl/colorizer-gl.cpp

```diff
@@ -290,64 +288,68 @@ namespace librealsense
                     }
                 }
                 
-                uint32_t output_rgb;
-                gf->get_gpu_section().output_texture(0, &output_rgb, TEXTYPE_RGB);
-                glBindTexture(GL_TEXTURE_2D, output_rgb);
-                glTexImage2D(GL_TEXTURE_2D, 0, GL_RGB, _width, _height, 0, GL_RGB, GL_UNSIGNED_BYTE, nullptr);
-                glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
-                glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
-
-                gf->get_gpu_section().set_size(_width, _height);
-
-                glBindFramebuffer(GL_FRAMEBUFFER, _fbo->get());
-                glDrawBuffer(GL_COLOR_ATTACHMENT0);
-
-                glBindTexture(GL_TEXTURE_2D, output_rgb);
-                _fbo->createTextureAttachment(output_rgb);
-
-                _fbo->bind();
-                glClearColor(1, 0, 0, 1);
-                glClear(GL_COLOR_BUFFER_BIT);
-
-                auto& shader = (colorize_shader&)_viz->get_shader();
-                shader.begin();
-                float max = _max;;
-                float min = _min;;
-                if (disparity)
+                if( auto gf = dynamic_cast< gpu_addon_interface * >( (frame_interface *)res.get() ) )
```

Comment:
Just to make sure, please validate colorizer works as expected when GLSL for processing is turned on with this change

Reply:
Of course... I activated GLSL for processing flag, checked 3D view and that textures are shown as expected